src/actions/summit-actions.js

```diff
@@ -74,8 +73,112 @@ export const getSummitById = (summitId) => async (dispatch) => {
       "badge_types," +
       "badge_features_types," +
       "badge_access_level_types," +
-      "badge_view_types," +
-      "tax_types"
+      "badge_view_types",
+    fields:
+      "id," +
+      "name," +
+      "active," +
+      "allow_update_attendee_extra_questions," +
+      "available_on_api," +
+      "calendar_sync_desc," +
+      "calendar_sync_name," +
+      "dates_label," +
+      "end_date," +
+      "event_types,event_types.id,event_types.name,event_types.order" +
+      "link," +
+      "locations,locations.id,locations.name,locations.order,locations.class_name,locations.rooms,locations.floors," +
```

Comment:
can we get rid of the locations ?
this is a lot of data .. where is this used ?